src/rust/generators/sysmon-subgraph-generator/src/generator.rs

```diff
@@ -59,19 +94,26 @@ where
 
             let graph = match Graph::try_from(event.clone()) {
                 Ok(subgraph) => subgraph,
+                Err(SysmonGeneratorError::UnsupportedEventType(s)) => {
```

Comment:
might be worth metric'ing or logging

Reply:
Yeah, it logs internally:
                    error!("Graph::try_from failed with: {:?}", e);